blue-xdma/src/include/Config.defines

```diff
@@ -0,0 +1,37 @@
+typedef 3000 WAITRESET;
+typedef 10000 TIMEOUT;
+
+///////////////////////////////////////////////
+/////           AXI4 Slave                /////
+///////////////////////////////////////////////
+
+typedef 64 AXI4_SLAVE_ADDRSz;
+typedef 64 AXI4_SLAVE_DATASz;
+typedef 4 AXI4_SLAVE_IDSz;
+typedef 0 AXI4_SLAVE_USRSz;
+typedef 0 AXI4_SLAVE_FIFOSz;
+typedef AXI4_Read_Rq#(AXI4_SLAVE_ADDRSz, AXI4_SLAVE_IDSz, AXI4_SLAVE_USRSz) AXI4_SLAVE_READ_RQ;
```

Comment:
use camel case for types with parameters:
`typedef AXI4_Read_Rq#(AXI4_SLAVE_ADDRSz, AXI4_SLAVE_IDSz, AXI4_SLAVE_USRSz) Axi4SlaveReadReq;`

Reply:
This has been changed to the method form in the latest commit.